dir.traversal.targets

```diff
@@ -172,6 +172,29 @@
     <Error Condition="'$(MSBuildLastTaskResult)'=='false'" />
   </Target>
 
+  <Target Name="GenerateTestProjectJsons">
```

Comment:
For consistency and to try and avoid confusion with the other target you defined here (they only differ by an "s"). Can you rename this to GenerateAllTestProjectJsons?